src/components/node-list/node-list.js

```diff
@@ -86,18 +103,14 @@ const NodeList = ({
                 hideTracksWhenNotNeeded
               >
                 <Filters
-                  getGroupState={getGroupState}
                   groupCollapsed={groupCollapsed}
                   groups={groups}
                   isResetFilterActive={isResetFilterActive}
                   items={items}
-                  onGroupToggleChanged={onGroupToggleChanged}
-                  onItemChange={onItemChange}
-                  onItemClick={onItemClick}
-                  onItemMouseEnter={onItemMouseEnter}
-                  onItemMouseLeave={onItemMouseLeave}
-                  onResetFilter={onResetFilter}
-                  onToggleGroupCollapsed={onToggleGroupCollapsed}
+                  onGroupToggleChanged={handleGroupToggleChanged}
+                  onItemChange={handleFiltersRowClicked}
+                  onResetFilter={handleResetFilter}
+                  onToggleGroupCollapsed={handleToggleGroupCollapsed}
```

Comment:
[Nit] I know the PR is still in draft. If we want to have the components - SearchList, NodeListTree, Filters in this file, this file needs to be shifted to a UI section or needs to be renamed. As `node-list.js` suggests only the list of nodes we see in the `pipeline-sidebar`. Wdyt @Huongg

Reply:
Agree. Maybe we could rename as follows 

Parent component could be called Node Explorer, and within that we have SearchList, NodeListTree, and Filters -- all of these could be top level components, i think both SearchList and Fitlers and NodeListTree can be too.

Reply:
hey @rashidakanchwala  would `NodeListManager` be a suitable name since it manages `NodeListTree`, `SearchList`, and `Filters`